docs/release_process.md

```diff
@@ -135,3 +136,33 @@ We're happy to announce that [AWX version 21.1.0](https://github.com/ansible/awx
 We're happy to announce that [AWX Operator version 0.22.0](https://github.com/ansible/awx-operator/releases/tag/0.22.0) is now available!
 
 ## Send the same IRC message (less the @newsbot) to #awx:ansible.com
+
+## Revert a Release
+
```

Comment:
Should we give examples of when we might want/not to revert a release?
ie when to revert, and when to make a new release instead?

Reply:
i just have a blurb saying "if you need to revert" - i guess it's up to the developer and/or awx team to give the thumbs up/down on a revert. what do you think @gundalow ?